samples/peripheral/802154_sniffer/README.rst

```diff
@@ -0,0 +1,152 @@
+
+LED 4:
+   When the sniffer captures a packet the LED is toggled on and off.
+
+Serial commands list
+********************
+
+This section lists the serial commands that are supported by the sample.
+
+channel - Change the radio channel
+==================================
+
+The command changes the IEEE 802.15.4 radio channel to listen on.
+
+   .. parsed-literal::
+      :class: highlight
+
+      channel *<channel>*
+
+The ``<channel>`` argument is an integer in the range between 11 and 26.
+
+For example:
+
+   .. parsed-literal::
+      :class: highlight
+
+      channel *23*
+
+receive - start capturing packets
+=================================
+
+The sniffer enters RX state and starts capturing packets.
+
+   .. parsed-literal::
+      :class: highlight
+
+      receive
+
+The received packets will be printed to the command-line with the following format:
+
+   .. parsed-literal::
+      :class: highlight
+
+      received: *<data>* power: *<power>* lqi: *<lqi>* time: *<timestamp>*
+
+* The ``<data>`` is a hexidecimal string representation of the received packet.
+* The ``<power>`` value is the signal power in dBm.
+* The ``<lqi>`` value is the IEEE 802.15.4 Link Quality Indicator.
+* The ``<timestamp>`` value is the absolute time of the received packet since the sniffer booted.
+
+sleep - stop capturing packets
+==============================
+
+Disables the radio and ends the receive process.
+
+   .. parsed-literal::
+      :class: highlight
+
+      sleep
+
+Configuration
+*************
+
+|config|
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/peripheral/802154_sniffer`
+
+.. include:: /includes/build_and_run.txt
+
+.. _802154_sniffer_testing:
+
+Testing the sample
+==================
+
+After programming the sample to your development kit, complete the following steps to test it:
+
+1. Connect the development kit to the computer using a USB cable.
+   Use the development kit's nRF USB port (J3).
+   The kits are assigned a COM port (in Windows) or a ttyACM device (in Linux), visible in the Device Manager or in the :file:`/dev` directory.
+#. |connect_terminal|
+#. Switch to a radio channel with an ongoing radio traffic:
+
+   .. parsed-literal::
+      :class: highlight
+
+      channel *23*
+
+#. Start the capture process:
+
+   .. parsed-literal::
+      :class: highlight
+
+      receive
+
+   The LED_1 will start blinking with shorter intervals.
+
+#. If there is radio traffic on the selected channel, the sniffer should print the captured packets:
+
+   .. parsed-literal::
+      :class: highlight
+
+      received: 49a85d41a5fffff4110f10270000369756e65619d09428a04b301951821db234460aa5ec4ff506631ef8adb22674683700 power: -39 lqi: 220 time: 15822687
+
+   The LED_4 will toggle its state when a frame is received.
+
+#. Disable the capture:
+
+   See the following example for how to light up LED 1 on the nRF5340DK:
```

Comment:
```suggestion
   See the following example for how to light up **LED_1** on the nRF5340DK:
```

Reply:
Removed the line. It was copied from a different sample and not applicable to the 802.15.4 sniffer.